src/test/java/com/revature/batchservice/tests/BatchControllerTest.java

```diff
@@ -0,0 +1,67 @@
+package com.revature.batchservice.tests;
```

Comment:
need more thorough tests like validating dates and the like

Reply:
Newest commit implements your changes.